e2e/menu_items_block/addresses.spec.js

```diff
@@ -0,0 +1,83 @@
+import { test, expect } from "@playwright/test";
+import { VISUAL_EDITOR_URL } from "../helpers/constants.js";
+
+test.beforeEach(async ({ page }) => {
+  await page.goto(VISUAL_EDITOR_URL);
+});
+
+test("renders address menu items", async ({ page }) => {
+  await page.getByText("$A", { exact: true }).click();
+  await expect(page.locator(".menubar")).toBeVisible();
+  const visibleMenuButtons = [
+    "B",
+    "H2",
+    "H3",
+    "“”",
+    "<>",
+    "$A",
+    "$CTA",
+    "$C",
+    "$E",
+    "^",
+    "%",
+    "1.",
+    "-",
+  ];
+  const disabledMenuButtons = ["p"];
+
+  for (const button of visibleMenuButtons)
+    await expect(page.getByText(button, { exact: true })).toBeVisible();
+  for (const button of disabledMenuButtons)
+    await expect(page.getByText(button, { exact: true })).toBeDisabled();
+});
+
+test("loads addresses from the index file in the editor", async ({ page }) => {
+  await expect(
+    page.locator("#editor .address").getByText("HM Revenue and Customs"),
+  ).toBeVisible();
+});
+
+test("should render address in the editor on multiple lines clearing on double enter when clicking on '$A' and typing", async ({
```

Comment:
Checking the double return exits the block feels a little like testing Prosemirror to me

Reply:
Agreed to leave this in place as we are treating feature tests as product specifications and ignoring our implementation